pkg/core/config/main_test.go

```diff
@@ -231,7 +231,7 @@ var (
 			ID: "3",
 			Config: Config{
 				Spec: Spec{
-					Name: `{{ pipeline "Source.kindd" }}`,
+					Name: `{{ pipeline "Source.kind_" }}`,
```

Comment:
I believe this is just an item intended to not match any valid item, but especially `kind`. Using `kind_` should have the same effect as using `kindd` with the advantage of being clear that it's an intentional divergence from `kind`.